From 91942786a443b0faca27ad08373ffb61e5c2cb05 Mon Sep 17 00:00:00 2001
From: Yotam Ben Dosa <ybendosa@habana.ai>
Date: Mon, 12 Oct 2020 16:46:41 +0300
Subject: [PATCH] [SW-21835] openbmc: install: recovery: bypass mgr

Bypass the software manager when deploying the secondary chip image.
Since the version validation performed by the software manager is only
relevant for the main chip, there is no reason to do the same validation
for the secondary chip's image version.

Change-Id: I9a6b817a6a284a97c972469a1b41f25db1887cbc
Signed-off-by: Yotam Ben Dosa <ybendosa@habana.ai>
---

diff --git a/package/openbmc/install-hls-curr.sh b/package/openbmc/install-hls-curr.sh
index a369da2..971efbe 100755
--- a/package/openbmc/install-hls-curr.sh
+++ b/package/openbmc/install-hls-curr.sh
@@ -120,9 +120,7 @@
 
 	tar xf ${PWD}/obmc-phosphor-image-hls.static.mtd.all.tar
 	sshpass -p $BMC_ROOT_PASSWORD scp ${SSH_FLAGS} \
-		${PWD}/image-bmc root@$BMC_ADDRESS:/tmp/
-	sshpass -p $BMC_ROOT_PASSWORD ssh root@$BMC_ADDRESS ${SSH_FLAGS} \
-		"cp /tmp/image-bmc /run/initramfs/image-bmc-recovery"
+		${PWD}/image-bmc root@$BMC_ADDRESS:/run/initramfs/image-bmc-recovery
 }
 
 function verify_partition_size {
